docker/Dockerfile

```diff
@@ -62,6 +62,21 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 RUN /usr/local/bin/recursive_vcs_import.py src src/upstream
 
+# remove blacklisted packages from workspace
+ARG BLACKLISTED_PACKAGES_FILE="docker/blacklisted-packages.txt"
+ARG ENABLE_RECURSIVE_BLACKLISTED_PACKAGES="false"
+RUN echo "colcon list -p --base-paths src/ --packages-select \\" >> $WORKSPACE/.remove-packages.sh && \
```

Comment:
What happens if a package is blacklisted but not found by colcon list? Error?

Reply:
I think during `rm -rf` there will be an error because `colcon list` returned nothing.
We should catch this error.